src/api/getArticleMediaInfo.js

```diff
@@ -26,9 +26,19 @@ export const getArticleMediaInfo = (lang, title, fromCommon) => {
       (typeof ImageDescription.value === 'string' && ImageDescription.value) ||
       (ImageDescription.value[lang] || ImageDescription.value[Object.keys(ImageDescription.value)[0]])
     )
+    const descriptionDir = ImageDescription && (
+      (typeof ImageDescription.value === 'string' && 'ltr') ||
```

Comment:
what I noticed, normally it returns the language in `en`, therefore `ltr` here

Reply:
Thanks for the useful feedback from the team, and I also notice here doesn't truly return `en` as default, that can be any language, thanks to the support of `<bdi>`, it shows the correction direction even it is rtl in english (means still text-align right but readable due to bdi. Therefore `descriptionDir` can be removed.